packages/rooks/package.json

```diff
@@ -7,6 +7,15 @@
   "unpkg": "dist/umd/rooks.umd.js",
   "types": "dist/types/index.d.ts",
   "typings": "dist/types/index.d.ts",
+  "exports": {
+    ".": {
+      "types": "./dist/types/index.d.js",
+      "module": "./dist/esm/index.js",
```

Comment:
Kept the `module` for webpack support (it's not part of the final spec). Not sure if recent versions needs that.

Didn't add the `browser` field (umd), cause nextjs 12.x will pick it up and defeat tree-shakability (browser bundle is one file)

Reply:
@belgattitude Could you help me with any reading material for me to fully understand what has changed in the `exports` field? It changes so often I lost track of it. Just want to make sure I understand all the changes we are making and we don't accidentally break anything.

Reply:
Most up-to-date I found out is

https://github.com/frehner/modern-guide-to-packaging-js-library#define-your-exports

Reply:
If you feel unsure I can cherry-pick this and create another PR.

Reply:
@belgattitude Thanks for this! I will read this and get back. I will take a good look at this PR tomorrow.

Reply:
TIL learnt a lot from that link today. Thanks!